frontend/src/app/globals.css

```
@@ -42,7 +35,8 @@ input[type="number"]:hover {
 }
 
 .Inter {
-  font-family: ui-sans-serif, system-ui, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
+  font-family: ui-sans-serif, system-ui, sans-serif, Apple Color Emoji,
+    Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
```

Biome

[error] Duplicate font names are redundant and unnecessary: Emoji

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)